assets/questions/mostCommonChar.js

```diff
@@ -26,21 +26,41 @@ globalData.questions['common'] = {  // eslint-disable-line dot-notation
   },
   auxiliaryCode: {
     python:
-`class AuxiliaryCode(object):
+`from collections import Counter
+class AuxiliaryCode(object):
     @classmethod
-    def foo(cls, word):
-        return None
+    def lettersOnly(cls, word):
+      if word == "":
+        return ""
+      word = [w for w in word if w.isalpha()]
+      counter = Counter(word)
+      result = word[0]
+      for w in counter:
+        if counter[w] > counter[result]: result = w
+      return w
+
+    @classmethod
+    def lowercaseOnly(cls, word):
+      if word == "":
+        return ""
+      word = [w for w in word if w.islower()]
+      counter = Counter(word)
+      result = word[0]
+      for w in counter:
+        if counter[w] > counter[result]: result = w
+      return w
 `
   },
   tasks: [{
     instructions: [
       [
-        'Find the most common character of a string, as described in title. ',
-        "Assume that the string is ASCII, and there's only one most common character. ",
-        'Consider that the answer is unique for now.'
+        "For this problem, we'd like you to write a method to determine",
+        'the most common character in a string You will be given a string',
+        'of ASCII characters, and you may assume that there',
+        ' is only one most common character, for now.'
       ].join(''),
       [
-        'We will guarantee that you will always be passed a string, so ',
+        "There's no need to validate that you're always passed a string",
         "you don't have to take care of integrity check."
```

Comment:
Drop this line. Make this part just "There's no need to validate that you're always passed a string."